Automattic/wp-calypso
# Global edge cache: fix toggle state for private and coming soon states

Closes https://github.com/Automattic/wp-calypso/issues/94130

## Proposed Changes

This PR shows the toggle for the Global cache as toggled off in the disabled state.

## Why are these changes being made?


Before

<img width="1235" alt="Screenshot 2024-09-05 at 1 42 49 PM" src="https://github.com/user-attachments/assets/cef6d109-290b-4b46-aa3e-adcee7775419">

After

<img width="1236" alt="Screenshot 2024-09-05 at 1 43 04 PM" src="https://github.com/user-attachments/assets/aea67436-ce56-4350-a81c-ea352b5943c8">

## Testing Instructions



* Navigate to Calypso Live link or pull the changes from this branch
* Make sure that you have a site either in private mode or in coming soon mode
* Navigate to `overview/{yourSite}`
* Navigate to `Server settings` tab
* Observe `Cache card`
* Confirm that the edge cache toggle is not toggled on
* Navigate to `settings/general/{yourSite}`
* Make the site public
* Navigate to `overview/{yourSite}`
* Navigate to `Server settings` tab
* Observe `Cache card`
* Confirm that the toggle is active and you can manipulate it

## Pre-merge Checklist
